themes/codeforpoland/views/partials/footer.jade

```diff
@@ -1,16 +1,14 @@
 footer
-  .container
-    .row.partners-title
-      .col-sm-12
-        h3 Partners
-    .row.partners-logos
-      .col-sm-12.clearfix
-        each sponsor in brigade.sponsors.other
-          a(href=sponsor.link)
-            img(src=sponsor.image, alt=sponsor.name)
-    //- .row.show-partners
-    //-   .col-sm-12
-    //-     a.btn.btn-danger(href='http://codeforpoland.org/partnerzy/') Show all partners
+  if (view=='home')
```

Comment:
ack, I wanted to just work on stuff and learn how things are related. No worries reverting code.

From an aesthetic purpose, the most prominent place we can place the sponsors is on the front page. For other pages, which include business info, it's extra information. Like if I was on the events page, I don't want to see the sponsors, I just want to see the events.